### ISSUE TITLE
serialization and julia versions

### ISSUE THREAD

**Issue by gdkrmr** [JuliaTime/TimeZones.jl#309]
I have installed `TimeZones.jl` with the latest Julia master and then installed it again with Julia 1.5.3 and got the following error in Julia 1.5.3:

```julia
ERROR: LoadError: Cannot read stream serialized with a newer version of Julia.
Got data version 13 > current version 10
```

I had to rebuild the package in order to make it work again. I don't think this is ideal, maybe you should not use the Julia internal serialization but a different format.

**Comment by nickrobinson251** [comment]
This is a duplicate: it's tracked at https://github.com/JuliaTime/TimeZones.jl/issues/288

(and yeah, this issue is annoying...)

**Comment by omus** [comment]
Should be fixed in TimeZones 1.5.3
